test/ng/parseSpec.js

```diff
@@ -3232,6 +3232,18 @@ describe('parser', function() {
           expect(fn()).toEqual(undefined);
         }));
 
+        it('should invoke a stateless filter once when the parsed expression has an interceptor',
+           inject(function($parse, $rootScope) {
+          var countFilter = jasmine.createSpy();
+          var interceptor = jasmine.createSpy();
+          countFilter.and.callThrough();
```

Comment:
In a strict sense, no, but there is some special logic for one time binding and when something is evaluated to `undefined` and I want to be sure that whoever reads the test, that we are not running into that case.